doc/sundials_developers/source/conf.py

```diff
@@ -11,238 +9,194 @@
-
-\makeatother
-
-\clearpage
-
-% Disclaimer
-\thispagestyle{empty}% no number of this page
-\vglue5\baselineskip
-\begin{center}
-  {\bf DISCLAIMER}
-\end{center}
-\noindent
-This document was prepared as an account of work sponsored by an agency of
-the United States government. Neither the United States government nor
-Lawrence Livermore National Security, LLC, nor any of their employees makes
-any warranty, expressed or implied, or assumes any legal liability or responsibility
-for the accuracy, completeness, or usefulness of any information, apparatus, product,
-or process disclosed, or represents that its use would not infringe privately owned rights.
-Reference herein to any specific commercial product, process, or service by trade name,
-trademark, manufacturer, or otherwise does not necessarily constitute or imply its endorsement,
-recommendation, or favoring by the United States government or Lawrence Livermore National
-Security, LLC. The views and opinions of authors expressed herein do not necessarily state
-or reflect those of the United States government or Lawrence Livermore National Security, LLC,
-and shall not be used for advertising or product endorsement purposes.
-
-\vskip2\baselineskip
-\noindent
-This work was performed under the auspices of the U.S. Department of Energy by Lawrence
-Livermore National Laboratory under Contract DE-AC52-07NA27344.
-\vfill
-\begin{center}
-  Approved for public release; further dissemination unlimited
-\end{center}
-
-\clearpage
-
-% Contributors
-\thispagestyle{empty}% no number of this page
-\vglue5\baselineskip
-\begin{center}
-{\bf CONTRIBUTORS}
-\end{center}
-\noindent
-The SUNDIALS library has been developed over many years by a number of
-contributors. The current SUNDIALS team consists of Cody J. Balos,
-David J. Gardner, Alan C. Hindmarsh, Daniel R. Reynolds, and
-Carol S. Woodward. We thank Radu Serban for significant and critical past
-contributions.\\
-\vskip 2em%
-\noindent
-Other contributors to SUNDIALS include: James Almgren-Bell, Lawrence E. Banks,
-Peter N. Brown, George Byrne, Rujeko Chinomona, Scott D. Cohen, Aaron Collier,
-Keith E. Grant, Steven L. Lee, Shelby L. Lockhart, John Loffeld, Daniel McGreer,
-Slaven Peles, Cosmin Petra, H. Hunter Schwartz, Jean M. Sexton,
-Dan Shumaker, Steve G. Smith, Allan G. Taylor, Hilari C. Tiedeman, Chris White,
-Ting Yan, and Ulrike M. Yang.
-\clearpage
-
-% clear empty double page
-\newpage{\pagestyle{empty}\cleardoublepage}
-
-% Start arabic numbering
-\pagenumbering{arabic}
-
-% =====================================================
-% End custom cover page (see latex_elements in conf.py)
-% =====================================================
-'''
+# Override format strings that numref/numfig uses
+numfig_format = {
+  'section': '§%s'
 }
 
-
-# -- Options for HTML output -------------------------------------------------
+rst_prolog = open('../../shared/global.rst.txt', 'r').read()
+
+rst_epilog = """
+.. |YEAR| replace:: 2021
```

Comment:
Should this be 2022?

Reply:
Yes, Cody has addressed this in #152. The year will now get automatically updated by the version update script.